src/tools/rados/rados.cc

```diff
@@ -476,11 +649,16 @@ static int do_append(IoCtx& io_ctx, RadosStriper& striper,
       cerr << "error reading input file " << infile << ": " << cpp_strerror(ret) << std::endl;
       goto out;
     }
+
+#ifndef WITH_LIBRADOSSTRIPER	
```

Comment:
This `#ifndef` appears to be a noop because it's inside `#else // WITH_LIBRADOSSTRIPER`